src/comparisonData.tsx

```diff
@@ -7,20 +7,11 @@ export type ComparisonData = {
 
 export interface Comparison {
   id: number,
-  category: ComparisonCategory,
+  category: ComparisonCategory | undefined,
```

Comment:
We should look into this to make sure this is ok - it feels strange to type a variable as undefined.